processor/add_orchestrator_metadata/add_orchestrator_metadata.go

```diff
@@ -27,19 +28,24 @@ import (
 	agentconfig "github.com/elastic/elastic-agent-libs/config"
 )
 
+const (
+	processorName  = "add_cluster_id"
+	clusterNameKey = "orchestrator.cluster.name"
+	ClusterIdKey   = "cluster_id"
+)
+
 func init() {
-	processors.RegisterPlugin("add_cluster_id", New)
-	jsprocessor.RegisterPlugin("AddClusterID", New)
+	processors.RegisterPlugin(processorName, New)
+	jsprocessor.RegisterPlugin("AddEnvironmentMetadata", New)
```

Comment:
didn't we decide to keep the old processor name?

Reply:
I probably missed it when I reverted it to the old name - I fixed it now